### PR TITLE
[Merged by Bors] - Add a flag to disable peer scoring

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

N/A

## Proposed Changes

Adds a flag for disabling peer scoring. This is useful for local testing and testing small networks for new features.

### CODE REVIEW

**Comment by pawanjay176** [comment]
Works as expected in local testnets.

File: beacon_node/src/cli.rs [hunk 1]
```diff
@@ -233,6 +233,13 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .help("Disables the discv5 discovery protocol. The node will not search for new peers or participate in the discovery protocol.")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("disable-peer-scoring")
+                .long("disable-peer-scoring")
+                .help("Disables peer scoring in lighthouse. WARNING: This is a dev only flag is only meant to be used in local testing scenarios \
+                        Using this flag on a real network may cause your node to become eclipsed and see a different view of the network")
+                .takes_value(false),
+        )
```
Comment (divagant-martian) [inline beacon_node/src/cli.rs:243]: since this is a dev flag we should probably mark it as hidden

**Review by divagant-martian — COMMENTED** [review]
Minor suggestion but otherwise LGTM

File: beacon_node/src/cli.rs (hunk 1 above)
Comment (pawanjay176) [reply beacon_node/src/cli.rs:243]: Fixed!

**Comment by divagant-martian** [comment]
conflicts arose. After that I think we can ship this. Since this is disabled by default I think this does not conflict with #4159

**Comment by pawanjay176** [comment]
@divagant-martian any blocker for this?

**Review by divagant-martian — APPROVED** [review]

**Pull request closed without merging**